src/odemis/acq/test/fastem_test.py

```diff
@@ -109,6 +115,362 @@ def test_overview_acquisition(self):
+        res_x, res_y = self.multibeam.resolution.value  # single field size
+        px_size_x, px_size_y = self.multibeam.pixelSize.value
+
+        # FIXME: It is not clear in which coordinate system the coordinates of the ROA are!!!
+        # Note: the coordinates are in the stage coordinate system with role='stage' and not role='stage-scan'.
+        # However, for fast em acquisitions we use stage-scan, which scans along the multiprobe axes.
+        # FIXME: This test does not consider yet, that ROA coordinates need to be transformed into the
+        #  correct coordinate system. Replace role='stage' with role='stage-scan' when function available.
+        # Note: Do not change those values; calculate_field_indices handles floating point errors the same way
+        # as an ROA that does not match an integer number of field indices by just adding an additional row or column
+        # of field images.
+        top = -0.002  # top corner coordinate of ROA in stage coordinates in meter
+        left = +0.001  # left corner coordinate of ROA in stage coordinates in meter
+        coordinates = (top, left, top + res_x * px_size_x * x_fields, left + res_y * px_size_y * y_fields)  # in m
+        roc = fastem.FastEMROC("roc_name", coordinates)
+        roa_name = time.strftime("test_megafield_id-%Y-%m-%d-%H-%M-%S")
+        roa = fastem.FastEMROA(roa_name,
+                               coordinates,
+                               roc,
+                               self.asm,
+                               self.multibeam,
+                               self.descanner,
+                               self.mppc
+                               )
+        path_storage = os.path.join(datetime.today().strftime('%Y-%m-%d'), "test_project_stage_move")
+
+        f = fastem.acquire(roa, path_storage, self.multibeam, self.descanner, self.mppc, self.stage)
+        data, e = f.result()
+
+        self.assertIsNone(e)  # check no exceptions were returned
+
+        # total expected stage movement in x and y during the acquisition
+        # half a field to start at center of first field image
+        exp_move_x = res_x/2. * px_size_x + res_x * px_size_x * (x_fields - 1)
+        exp_move_y = res_y/2. * px_size_y + res_y * px_size_y * (y_fields - 1)
+
+        # TODO these comments are true, when stage is replaced with stage-scan
+        # Move in the negative x direction, because the second field should be right of the first.
+        # Move in positive y direction, because the second field should be bottom of the first.
+        exp_position = (top - exp_move_x, left + exp_move_y)
+        # get the last stage position (it is the center of the last field)
+        cur_position = (self.stage.position.value['x'], self.stage.position.value['y'])
+
+        # check stage position is matching expected position (Note: stage accuracy is ~ TODO fix decimal accordingly)
+        numpy.testing.assert_almost_equal(exp_position, cur_position, decimal=6)
```

Comment:
do you need to move back the stage to the original position at the end of this test?

Reply:
No we don't - we also don't do this in the normal acquisition now, as we want to move to the next ROA instead. We did that back on the system only I think, so that we are back where we started. But that is a special use case I would say.